src/snowflake/connector/converter.py

```diff
@@ -366,8 +366,8 @@ def _nonetype_to_snowflake_bindings(self, *_) -> None:
         return None
 
     def _date_to_snowflake_bindings(self, _, value: date) -> str:
-        # milliseconds
-        return _convert_date_to_epoch_milliseconds(value)
```

Comment:
If this is the only reference to `_convert_date_to_epoch_milliseconds`, could we remove it?

Reply:
good callout, I think we can remove it